packages/embark-async-wrapper/package.json

```diff
@@ -21,6 +21,9 @@
     "url": "https://github.com/embark-framework/embark.git"
   },
   "main": "./dist/index.js",
+  "files": [
+    "dist"
```

Comment:
`"files"` is a whitelist and once it's in place only exactly what's in there (other than default includes, see link to npm docs in PR description) will be packaged into the tarball.

Reply:
So including the `dist/` folder and it's contents in the tarball prevents the package from needing to be built on the user's machine during install?

Reply:
Yes, and because we're using a whitelist if other meta dot files or whatever (that don't need to be published but are part of monorepo life) are added to to the package, they won't inadvertently end up in the tarball.